compsoc-constitution.tex

```diff
@@ -1,80 +1,53 @@
-% Preamble
-% ---
 \documentclass{article}
 
-% Packages
-% ---
+\usepackage[T1]{fontenc}
+\usepackage[british]{babel}
+\usepackage{microtype}
+\usepackage{csquotes}
 
-\usepackage{geometry}
-\usepackage{textcomp}
-
-% Commands
-% --
-
-\newcommand*{\SignatureAndDate}[1]{%
-    \par\noindent\makebox[2.5in]{\hrulefill} \hfill\makebox[2.0in]{\hrulefill}%
-    \par\noindent\makebox[2.5in][l]{#1}      \hfill\makebox[2.0in][l]{Date}%
-}%
-
-\geometry{
-  a4paper,
-  total={210mm,297mm},
-  left=20mm,
```

Comment:
In general I'm a proponent of the default LaTeX margin sizes because it makes the textblock width a little better for readability (The Elements of Typographic Style by Robert Bringhurst has a good explanation). But if you do want the smaller margins, that's actually the default for `\usepackage[a4paper]{geometry}` so that's all we need.